Missing docs for informative-docs > excludedTags option

## Motivation

`jsdoc/informative-docs` will scan the body and all tags of a JSDoc tag for informative-ness. This is good and intended in most cases. But sometimes, there are tags that are used as categorization / metadata and only coincidentally match the name of the construct they're documenting. For example:

```js
/**
 * Calculates and retrieves the compiler options for the current project.
 * @category Compiler Options
 */
function getCompilerOptions() {
  // ...
}
```

Standalone reproduction of that snippet for reference: https://github.com/JoshuaKGoldberg/repros/tree/jsdoc-informative-docs-category-tag

## Current behavior

```plaintext
  3:1  error  This tag description only repeats the name it describes  jsdoc/informative-docs
```

## Desired behavior

Can we add an option to the rule to allow a list of tags to be ignored? Say, `ignoredTags`?

```js
"jsdoc/informative-docs": ["error", {
  "ignoredTags": ["category"]
}]
```

## Alternatives considered

An inline disable comment is doable, but clutters the code and is generally inconvenient.

--- a/.README/rules/informative-docs.md
+++ b/.README/rules/informative-docs.md
@@ -35,6 +35,20 @@ The default `aliases` option is:
 }
 ```
 
+### `excludedTags`
+
+Tags that should not be checked for valid contents.
+
+For example, with `{ excludedTags: ["category"] }`, the following comment would not be considered uninformative:
+
+```js
+/** @category Types */
+function computeTypes(node) {
+  // ...
+}
+
+No tags are excluded by default.
+
 ### `uselessWords`
 
 Words that are ignored when searching for one that adds meaning.

--- a/docs/rules/informative-docs.md
+++ b/docs/rules/informative-docs.md
@@ -47,6 +47,22 @@ The default `aliases` option is:
 }
 ```
 
+<a name="user-content-informative-docs-options-excludedtags"></a>
+<a name="informative-docs-options-excludedtags"></a>
+### <code>excludedTags</code>
+
+Tags that should not be checked for valid contents.
+
+For example, with `{ excludedTags: ["category"] }`, the following comment would not be considered uninformative:
+
+```js
+/** @category Types */
+function computeTypes(node) {
+  // ...
+}
+
+No tags are excluded by default.
+
 <a name="user-content-informative-docs-options-uselesswords"></a>
 <a name="informative-docs-options-uselesswords"></a>
 ### <code>uselessWords</code>

